corehq/apps/programs/forms.py

```diff
@@ -11,6 +13,9 @@ def __init__(self, program, *args, **kwargs):
 
         kwargs['initial'] = self.program._doc
         super(ProgramForm, self).__init__(*args, **kwargs)
+        self.helper = FormHelper()
```

Comment:
make sure to add `self.helper.form_tag = False` here because I think crispy forms is closing the form tag prematurely (form actions appears outside of the form, meaning that the submit button has no effect)